source/_posts/azure-purview.md

```diff
@@ -0,0 +1,149 @@
+
+## Business Context, Classification & Lineage
+
+Once the scan is complete, the data users in your company can start using the data that was just scanned. One thing they can do is add business context to the different fields
+
+<img src="/images/azure-purview/business-context.png" />
+
+In the example you can see that you can expand on the data within your organistation by adding descriptions and context to them. This allows an organisation to define any domain knowledge or business context to their data users in a clear way. 
+
+Secondly, Classification is another important part of Azure Purview. It's important for an organisation to be compliant with rules and regulations like GDPR. Azure Purview helps with this by classifying data fields. It uses AI to see what a field contains, and classifies it accordingly. With this knowledge, you know where within your company your sensitive data resides. 
+
+Last but not least, Azure Purview offers Data Lineage. Data Lineage lets you see where your data comes from and where you're data is going to. The image below shows the flow of data from the salesOrderHeader table to different areas within the organisation. Some operations are applied to the data, which is then transformed and shown in PowerBI. 
+
+<img src="/images/azure-purview/lineage.png" />
+
+Data Lineage in combined with Classification can be really helpful to see where sensitive data is used and whether it should be available in those specific places. 
+
+# **How to get started with Azure Purview**
+
+To get started, you first need to make sure you are the administrator for your subscription. It is also important to note that you need to enable specific resources for the subscription to be able to use Azure Purview:
+
+- Microsoft.Purview
+- Microsoft.EventHub
+- Microsoft.Storage
+
+*Not having these enabled could cause some problems down the line.*
+
+Once these are enabled, Click on "Create a resource", search for Azure Purview to get started!
+
+<img src="/images/azure-purview/create-resource.png" />
+
+The Basic tab is pretty straightforward. Provide a valid resource group and name for the Purview account and select the appropriate location suitable for your usage.
+
+<img src="/images/azure-purview/configuration.png" />
+
+In the configuration tab, there are a few new options. As Azure Purview is still in preview, not much is known about what the platform sizing, catalog options and/or data insights settings mean (the learn more button only points to the pricing). 
+
+But, knowing that Azure Purview focuses on doing scans, platform size might have to do with scanning frequency and the number of scans you can perform. 
+
+Under the **catalog** category, there are two options (which are greyed out due to it being in preview). Here you can select which one is relevant for your implementation. The two options as of writing this blog are:
```

Comment:
here you add markup for **catalog**, check that you apply this markup consistently for similar terms (be consistent!)
greyed out >> grayed out (if using American spelling)